content/articles/angular-output-and-input-decorator/index.md

```diff
@@ -0,0 +1,198 @@
+2. @Input - A decorator that allows the parent to communicate with the child component.
+
+### Exploring Child component
+
+In this section, we're exploring how to create a link between the parent and the child component. Let's get started by following the instructions below:  
+
+1: Create a new component `musicDetailsComponent` by running the following command:
+
+```bash
+ng g component music-details
+```
+
+Output:
+
+```bash
+CREATE src/app/music-details/music-details.component.css (0 bytes)
+CREATE src/app/music-details/music-details.component.html (28 bytes)
+CREATE src/app/music-details/music-details.component.spec.ts (669 bytes)
+CREATE src/app/music-details/music-details.component.ts (302 bytes)
+UPDATE src/app/app.module.ts (422 bytes)
+
+```
+
+2: Update the `music-details.component.ts` with the following contents:
+
+```javascript
+// we import the `Input` decorator from Angular Core first.
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-music-details',
+  templateUrl: './music-details.component.html',
+  styleUrls: ['./music-details.component.css']
+})
+export class MusicDetailsComponent implements OnInit {
+
+  // in this line, we've a property MusiciaName declared as an Input.(refer to Input decorator role in the previous section)
+  @Input () musicianName: string | undefined;
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+}
+```
+
+As shown above, the `@Input` decorator is applied to the `musicianName` property, consequently decorating it. This implies that the value of the `musicianName` will be derived from the parent component.  
+
+3: Display the `musicianName` in the template
+
+```html
+<p>
+  The divide is the third studio album by the world-renown English singer, {{musicianName}}
+</p>
+
+```
+
+4: Using parent component
+Now that we've defined our `musicianName` property in the child component, let's see how we can pass it a value from the parent component.  
+
+In the `app.component.html` which is our parent component, add the following:
+
+```html
+<!-- We're using the child selectors in the parent component -->
+<app-music-details></app-music-details>
+```
+
+5: Assign musician a name in the parent component's `app.component.ts`
+
+```javascript
+-------------------------------
+export class AppComponent {
+  name = 'Edward Christopher Sheeran';
+}
+---------------------------------
+```
+
+Now that we've the musician name, the parent component passes this value to the child component as shown below using the `@Output` decorator:  
+
+```html
+<!-- update the app.component.ts file as shown below -->
+<app-music-details [musicianName]="name"></app-music-details>
+
+```
+
+In the above example, we pass the value of the `musicianName` using the parent `name` property which we defined in the parent component and assigned the value to.  
+
+Serve the application by running the following command:  
+
+```bash
+cd output-decorator
```

Comment:
I do not see the relevance of this command as there is no directory named `output-decorator`
Consider revising it.